cypress/specs/app.spec.ts

```diff
@@ -1,4 +1,4 @@
-const SNAPSHOT_DELAY = 100;
+const SNAPSHOT_DELAY = 500;
```

Comment:
Even with the retries ?

Reply:
The snapshot approval job was failing, because it would pass the first attempt regardless of whether it had properly rendered or not... I've been investigating a better solution, but I still can't find a reliable way to know if something a R3F component has actually rendered on the canvas or not...